cmd/http/main.go

```diff
@@ -97,8 +89,6 @@ func NewKeyValueHandler(conf config.KVConfiguration) *KeyValueHandler {
 }
 
 func (handler *KeyValueHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	handler.mu.Lock()
-	defer handler.mu.Unlock()
```

Comment:
It is related to isolation. Since we have not touched the isolation, it is OK to remove it for now.

Reply:
discussed offline. ok to defer isolation related design and impl later.

Reply:
as a safe check, please do a small perf test with this change to verify concurrent access. we'll kick off the isolation discussion/planning this week.